### ISSUE TITLE
New placeholder if game is running

### ISSUE THREAD

**Issue by Kirrasio** [ScreamingSandals/BedWars#150]
#### Placeholder to check if a specified game is running (not player specific) - 
Global Information
%bedwars_game_running[bedwars_name]%
Placeholder to check if any game is running
%bedwars_games_running%

Returns true or false

**Comment by SoCuul** [comment]
I'm pretty sure notfoundname made an addon adding these types of placeholders. Try it here: https://www.spigotmc.org/resources/addon-scbwa.76895/?__cf_chl_jschl_tk__=492f803d6cac7c2eb052eb1e961cd2cc0b0d9bba-1588217286-0-AWaIRgUnAanTi8aRexovIUQS68NtA2C7FtDzouY7C7GROfxcosFkS4i3Z3qleqg2tj6prrVgi0UsivdJc3aUbIKtl3VX5lPJ34xbO59wdGOhe-30LQIQiWsGdPU92xyX9F-TJLedrflyWgHi8ywAuYGHIeIfCRxpqV09g9LRD98Em2hBnP03C_appAo5P2nOaXN5zohs26ksAdkMu5kaBVjkSrEJ8qXbUOg9TMebWXaM_X0XeIyJ9lu9QL0ULhR0fYJxVplLuHJd8-F6G36cGikn3uPMoCBmeIrTHBVbYbI0I-1a9-rjcEFf2fI5biQvog

**Comment by Kirrasio** [comment]
I don't understand why these are not a part of the base plugin? I did install this just a bit ago and after trying it out it's not quite what I requested here. It returns the same waiting status even with no one in the lobby and no game running or pending. Example: With no game running or pending %scbwa_bedwars_state% returns "waiting", it also returns "waiting" after executing the command /bw join and joining the lobby. 
Basically I'm wanting to add some custom denizen scripts to execute commands before anyone joins the game lobby.

**Comment by iamceph** [comment]
Because we were too lazy to do that. Will be a thing in 1.x.x

**Comment by Kirrasio** [comment]
Ok, that's fair :) Thank you for responding. I was trying to figure out a way around the spawn in structures not getting reset, I figured I could run a series of reset commands to reset the world if I had a placeholder to check for the status of the game.

**Comment by iamceph** [comment]
There is still the possibility to do that via API :)

**Comment by Misat11** [comment]
New placeholders were added in snapshots of 0.2.33 providing this exact information in this specific format, this issue can now be closed.

- `%bedwars_game_<game>_running%`
- `%bedwars_game_<game>_waiting%`
- `%bedwars_current_game_running%`
- `%bedwars_current_game_waiting%`
- `%bedwars_all_games_anyrunning%`
- `%bedwars_all_games_anywaiting%`

**Comment by SoCuul** [comment]
> New placeholders were added in snapshots of 0.2.33 providing this exact information in this specific format, this issue can now be closed.
> 
>     * `%bedwars_game_<game>_running%`
> 
>     * `%bedwars_game_<game>_waiting%`
> 
>     * `%bedwars_current_game_running%`
> 
>     * `%bedwars_current_game_waiting%`
> 
>     * `%bedwars_all_games_anyrunning%`
> 
>     * `%bedwars_all_games_anywaiting%`

I can't believe you replied after like 4 years 😭

**Comment by Misat11** [comment]
was going through all the issues, some of them should have been closed by now, some of the issues were easily solvable (probably weren't back then). I see that the author of the issue probably doesn't care anymore, but I felt I should add a comment on why I decided to close the issue.
